src/ReactiveUI.sln

```diff
@@ -1,6 +1,7 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26730.15
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
```

Comment:
Do we want to change the Visual Studio version?

Reply:
I think it's fine yeah. We are going down the route of requiring it anyway for maintainers.